editor/filesystem_dock.cpp

```diff
@@ -3796,13 +3833,21 @@ FileSystemDock::FileSystemDock() {
 	button_toggle_display_mode->set_flat(true);
 	toolbar_hbc->add_child(button_toggle_display_mode);
 
+	button_dock_placement = memnew(Button);
+	button_dock_placement->set_flat(true);
+	button_dock_placement->set_tooltip_text(TTR("Change Dock Placement"));
```

Comment:
I found it odd that other placement buttons don't have tooltips 🤔

Reply:
Ah likely it ended up like this due to the refactor.